tests/manual-tests/components/reorderer/dynamic/css/reorderer.css

```diff
@@ -8,3 +8,8 @@
     background-color: pink;
     opacity: 0.75;
 }
+.fl-reorderer-dropMarker {
+    background-color: #F00;
+    height          : 0;
```

Comment:
line 13, 14: remove extra spaces before `:` to be consistent with the css convention elsewhere.